src/converse-controlbox.js

```diff
@@ -610,8 +610,18 @@ converse.plugins.add('converse-controlbox', {
                     }
                 }
             });
-        });
 
+        });
+        _converse.api.listen.on('chatBoxViewPortCalculate', () => {
+            const vh = window.innerHeight * 0.01;
+            document.documentElement.style.setProperty('--vh', `${vh}px`);
+        });
```

Comment:
Instead of triggering and listening for a new event, you can just listen for `chatBoxViewsInitialized`